test-commons/src/main/java/no/unit/nva/publication/PublicationGenerator.java

```diff
@@ -181,12 +181,12 @@ public static Identity randomIdentity() {
             .withId(randomUri())
             .withArpId(randomString())
             .withNameType(NameType.PERSONAL)
-            .withOrcId(randomString())
+            .withOrcId(UUID.randomUUID().toString())
             .build();
     }
 
     private static Set<AdditionalIdentifier> randomAdditionalIdentifiers() {
-        return Set.of(new AdditionalIdentifier(randomString(), randomString()));
+        return Set.of(new AdditionalIdentifier(FAKER.lorem().word(), randomUri().toString()));
```

Comment:
use randomString

Reply:
Using randomWord for key, kept randomUri as we need something witout space as identifier